website/content/en/preview/getting-started/getting-started-with-karpenter/scripts/step12-add-provisioner.sh

```diff
@@ -13,7 +13,8 @@ spec:
       cpu: 1000
```

Comment:
There's more notes about this value in the doc. Can you read through the doc and make sure that everything is consistent and makes sense with this new change?